src/app/App.vue

```diff
@@ -21,7 +21,7 @@
             data-testid="app-error"
           />
 
-          <AppOnboardingNotification v-if="!isWizard && shouldShowOnboardingNotification" />
+          <AppOnboardingNotification v-if="!isWizard && route.name" />
```

Comment:
**Question**: Why is the addition of `route.name` needed?

Reply:
Pretty cure its not necessary, I've removed it, thanks for the spot!